augurpy/plotting/differential_prioritization.py

```diff
@@ -0,0 +1,21 @@
+from matplotlib import pyplot as plt
+
+
+def plot_differential_prioritization(results, top_n=None):
```

Comment:
Usually we try to always make the matplotlib axes available as a parameter and a couple more (see e.g. what Scanpy's plots expose). This helps with custom plots where you e.g. want to create an image with several plots.

Would also return the axes here I guess.

Reply:
Ah, I'll look into that!